pkg/server/plugin/nodeattestor/x509pop/x509pop.go

```diff
@@ -35,15 +38,19 @@ func builtin(p *Plugin) catalog.BuiltIn {
 }
 
 type Config struct {
+	SPIRETrustBundle  bool     `hcl:"spire_trust_bundle"`
+	SPIFFEPrefix      *string  `hcl:"spiffe_prefix"`
```

Comment:
SVIDPrefix? Also, if we don't need to distinguish between this flag being unset and "" (e.g. if "" can safely imply unset), I'd suggest this be of type `string` to reduce pointery-ness.